src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests_Synthesized.cs

```diff
@@ -984,7 +984,7 @@ public StringBuilder Append(string s)
 
 ";
             var comp = CreateEmptyCompilation(source);
-            CompileAndVerify(comp, symbolValidator: validate);
+            CompileAndVerify(comp, symbolValidator: validate, verify: Verification.Skipped);
```

Comment:
![image](https://user-images.githubusercontent.com/31348972/153048933-f2ef16a2-076c-4829-8250-bfc6a154c09d.png)

Reply:
Instead of using an empty compilation, I'd recommend using `CreateCompilation` then `MakeTypeMissing`. Then I don't think we'll need to skip PE verification

Reply:
Yeah that type of PE verification failure is generally a real failure. It makes me worried that we missed a case and are attempting to emit a reference to the attribute when it doesn't exist. Think @jcouv approach is correct here to use `MakeTypeMissing` that will help rule out that problem.

Reply:
Nice! I didn't know `MakeTypeMissing` is a thing!